apis/fluent-en_GB/atrium-api-fluent-en_GB-common/src/main/kotlin/ch/tutteli/atrium/api/fluent/en_GB/anyAssertions.kt

```diff
@@ -30,22 +30,56 @@ fun <T> Expect<T>.notToBe(expected: T): Expect<T> = _logicAppend { notToBe(expec
 /**
  * Expects that the subject of the assertion is the same instance as [expected].
  *
+ * Deprecated as atrium moves to a consistent ‘to + infinitive’ naming convention. Use [toBeTheSameAs] instead.
+ * This function will be removed in version 1.0.0. See
+ * [atrium-roadmap#93](https://github.com/robstoll/atrium-roadmap/issues/93) for details and to give feedback.
+ *
  * @return An [Expect] for the current subject of the assertion.
  * @throws AssertionError Might throw an [AssertionError] if the assertion made is not correct.
+ */
+@Deprecated(
+    "Replaced by toBeTheSameAs. Will be removed with version 1.0.0",
```

Comment:
I plan to remove things maybe earlier than 1.0.0. They will certainly stay for a while but maybe we already remove it in 0.18.0 or 0.19.0
```suggestion
    "Replaced by toBeTheSameAs. Will be removed latest with version 1.0.0",
```

Reply:
Done for existing `@Deprecated` annotations